### ISSUE TITLE
A11Y: Safezone toggle should be a checkbox

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#4245]
## Bug Description

When using a screenreader and clicking the safe-zone toggle button, no information is given to the user, as this is "just" a button and not a real toggle. If it is instead implemented with a checkbox (like bold/italic/underline in the design panel), the user will be informed of the current state and changed state.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
